colortv/docs/androidTVSDK.md

```diff
@@ -511,3 +588,87 @@ public class MainActivity extends Activity {
+| ctv_hide                   | Any layout   | Visibility.GONE                     | Everything inside hide on focus                           | ALL    |
+| ctv_show                   | Any layout   | Visibility.VISIBLE                  | Everything inside show on focus                           | ALL    |
+| ctv_autoPlayTimerContainer | Any layout   |                                     | Used to inject countdown timer to autoplay first video    | ALL    |
+| ctv_videoPreviewContainer  | Any layout   |                                     | Play video preview. Automatically hide thumbnail on focus | ALL    |
+| ctv_ivVideoThumbnail       | ImageView    | Visibility.GONE                     | Display video thumbnail                                   | ALL    |
+| ctv_ivAppLogo              | ImageView    |                                     | Display logo                                              | ALL    |
+| ctv_ivPlayButton           | ImageView    | Visibility.GONE (on mobile - scale) |                                                           | ALL    |
+| ctv_ivBlackMask            | ImageView    | Visibility.GONE (on mobile - fade)  |                                                           | ALL    |
+| ctv_tvTitle                | TextView     |                                     | Display title                                             | ALL    |
+| ctv_tvDescription          | TextView     |                                     | Display description                                       | ALL    |
+| ctv_tvDuration             | TextView     |                                     | Display duration                                          | ALL    |
+| ctv_tvGenre                | TextView     |                                     | Define how each genre should looks like. Visibility must be GONE                   | ALL    |
+| ctv_llGenres               | LinearLayout |                                     | Each ctv_tvGenre will be added here                       | ALL    |
+
+```java
+#setRowCount(Device device, int rowCount)
+```
+Used to set number of rows for specified device type.
+
+```java
+#setFont(Device device, Typeface typeface)
+```
+Used to set custom font for specified device type. It works only if you don't use custom item layout.
+
+```java
+#setSnapEnabled(boolean enabled)`
+```
+Used to set snapping RecyclerView elements to the center of screen. Available only on Mobile. Value is forced to false if item layout is not default.
+
+- true - height of items is forced to be `MATCH_PARENT` and width is calculated by formula: `ITEM_HEIGHT*0.45`. If you change height of RecyclerView in custom grid layout we recommend to disable snapping. Video preview is enabled and will be shown if available.
+- false - height and width of items is set to be as defined in layout. Video preview is disabled and will be not shown whether it is available.
```

Comment:
`the height and width`

`the items`

`will be not` -> `will not be`

`whether it is avaliable or not`